src/PKSim.Infrastructure/Services/ImportObservedDataTask.cs

```diff
@@ -87,8 +87,9 @@ public ImporterConfiguration OpenXmlConfiguration()
 
       public void AddAndReplaceObservedDataFromConfigurationToProject(ImporterConfiguration configuration, IEnumerable<DataRepository> observedDataFromSameFile)
       {
+         var lstObservedDataFromSameFile = observedDataFromSameFile.ToList();
```

Comment:
Can you change the argument so it's `IReadOnlyList<DataRepository> observedDataFromSameFile` Then you won't have to create the list version with the funky name.

The method probably pre-dates the introduction of `IReadOnlyList` in .NET